articles/migrate/how-to-create-group-machine-dependencies.md

```diff
@@ -54,24 +51,20 @@ Attach a workspace as follows:
 
     ![Add workspace](./media/how-to-create-group-machine-dependencies/workspace.png)
 
-  - After you specify a name for a new workspace, it's created in the same geography as the Azure Migrate project.
-  - When you attach an existing workspace, you can pick from all the available workspaces in the same subscription as the migration project.
-  - You need Reader access to the workspace to be able to attach it.
-  - You can't modify the workspace associated with a project after it's attached. 
-
+    - After you specify a name for a new workspace, it's created in the same geography as the Azure Migrate project.
```

Comment:
This would be the main functional piece. Items under ordered lists need to be more than 3 spaces for them to render as child items